.github/workflows/cd.yml

```diff
@@ -1,13 +1,16 @@
 name: cd
 on:
   workflow_run:
-    workflows: ["ci"]
-    branches-ignore: ["*"]
+    workflows:
+      - ci
+    branches:
+      - master
+      - main
     types:
       - completed
   push:
     tags:
-      - "v*"
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Some plugins use build tags like in https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.1.6-rc1. Will `cd` still trigger for builds like that?

Reply:
No we also need to include the rc ones